src/nav/navbar.css

```diff
@@ -0,0 +1,31 @@
+@media (min-width: 500px) {

```

Comment:
? 2xNavbar component?

Reply:
am sters componenta care era in plus, fara PR, pt ca daca fac PR din main nu am nimic, am facut direct un PR mare cu toata aplicatia